plugins/coverbrowser.koplugin/covermenu.lua

```diff
@@ -232,7 +232,9 @@ function CoverMenu:updateItems(select_number)
     -- we replace it by ours.
     -- (FileManager may replace file_chooser.showFileDialog after we've been called once, so we need
     -- to replace it again if it is not ours)
-    if self.showFileDialog and self.showFileDialog ~= self.showFileDialog_ours then
+    if self.path -- FileManager only
+        and (not self.showFileDialog_ours -- never replaced
+              or self.showFileDialog ~= self.showFileDialog_ours) then -- it is no more ours
```

Comment:
no longer :-p

Reply:
Restored original wordings from https://github.com/koreader/koreader/pull/2940.

Reply:
It is a way @poire-z likes to put it. ;-P Anyway, as stated merging is fine by me, but it should be "no longer" ^_^

Reply:
(Yes, my English is a bit unstable on "no more" vs. "no longer" :))